### DISCUSSION TITLE
SAMSUNG GALAXY M01 CORE M013F/DS CUSTOM ROM & RECOV3RY CREATION<Your question or feedback about the GitHub Discussions product>

### DISCUSSION THREAD

**Discussion by KrishnaAJ** [community/community#131871] [Programming Help]
### Select Topic Area

Question

### Body

Can anyone create h custom rom & twrp recovery for samsung galaxy m01 core m013f/ds

**Comment by LiteBrite82** [comment]
Hi @KrishnaAJ ,

Thanks for being a part of the GitHub Community, we're glad you're here!

The Discussions category is for conversations related the GitHub product `Discussions`; sorry for the confusion. I have moved this post to `Programming Help`, however, it looks like you're looking for support from Samsung. You might want to try asking for help somewhere that specializes in Samsungs product line, such as seeing if they have a support page or forum.

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you're getting the best support you can and that your message goes where you intended, but this may not be the right place.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information. 

Best of luck!
